services/121-service/src/registration/registrations.service.ts

```diff
@@ -362,21 +323,22 @@ export class RegistrationsService {
     );
   }
 
-  public async importRegistrations(
+  public async importRegistrationsFromCsv(
     csvFile: Express.Multer.File,
     programId: number,
     userId: number,
   ): Promise<ImportResult> {
     const program = await this.findProgramOrThrow(programId);
-    return await this.registrationsImportService.importRegistrations(
+    this.throwIfProgramIsNotPublished(program.published);
```

Comment:
AFAIK we are removing the published attribute, so maybe no need to introduce a check for it anymore.

Reply:
Ah okay nice 👍. But now it's easier to remove right, because the error is not thrown in 2 places